x-pack/plugins/security_solution/server/endpoint/routes/resolver/tree/queries/descendants.ts

```diff
@@ -176,19 +190,21 @@ export class DescendantsQuery {
     nodes: NodeID[],
     limit: number
   ): Promise<FieldsObject[]> {
-    if (nodes.length <= 0) {
+    const validNodes = validIDs(nodes);
```

Comment:
Not allowing search requests for empty string IDs. This is also present at the POST request API schema level but adding here too just in case.